redash/tasks/queries.py

```diff
@@ -190,12 +191,20 @@ def refresh_queries():
             elif query.data_source.paused:
                 logging.info("Skipping refresh of %s because datasource - %s is paused (%s).", query.id, query.data_source.name, query.data_source.pause_reason)
             else:
-                if query.options and len(query.options.get('parameters', [])) > 0:
-                    query_params = {p['name']: p.get('value')
-                                    for p in query.options['parameters']}
-                    query_text = query.parameterized.apply(query_params).query
-                else:
-                    query_text = query.query_text
+                query_text = query.query_text
+
+                parameters = {p['name']: p.get('value') for p in query.options.get('parameters', [])}
+                if any(parameters):
+                    try:
+                        query_text = query.parameterized.apply(parameters).query
+                    except InvalidParameterError as e:
+                        logging.info("Skipping refresh of %s because of invalid parameters: %s", query.id, e.message)
```

Comment:
This and the other log message have the potential to significantly spam the log (in a busy system). Let's move them to `debug` for now.